### DISCUSSION TITLE
Problem Video DownloadHelper con mediaset infinity

### DISCUSSION THREAD

**Discussion by Caf77** [aclap-dev/video-downloadhelper#1574] [Language & Translation]
Hello or just purchased the license of Video DownloadHelper the problem is that on the site mesdiaset infinity https://mediasetinfinity.mediaset.it/video/ilmisterodellapietraazzurra/la-piccola-fuggiasca_F002213501000202 it detects the video downloads it but at the end of the download you can't see or hear the screen and it's all gray can you solve the problem thanks

**Comment by Mattmjs** [comment]
I have to think DRM is in use, it will indicate this only in Firefox. Chrome and Edge will not indicate a stream is using DRM but you've
not mentioned your browser in use. For your information a license will not help you on DRM streams, it's not a problem with the
software or the license. If you have Firefox look for this icon.

![DRM icon](https://github.com/user-attachments/assets/41309fa7-b883-4c52-83d2-e66eed7df7c0)
